src/Form/CustomFieldsFormService.php

```diff
@@ -113,7 +113,7 @@ public function addCustomFields($form, $dataClass = null, $overrideOptions = arr
                 $form->add($name, $fieldType, $options);
             }
             // add model transformer for entity type fields
-            if (EntityMapper::isEntityField($field['type'])) {
+            if (EntityMapper::isEntityField($field['type']) && $reverseAsString) {
```

Comment:
this is wrong as I found out, since the forward transformation is still required (otherwise, single entity fields do not work). Will be corrected in #22